src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/hosts/hosts.component.ts

```diff
@@ -30,6 +31,7 @@ export class HostsComponent implements OnInit {
     private cephShortVersionPipe: CephShortVersionPipe
   ) {
     this.permissions = this.authStorageService.getPermissions();
+    this.grafanaScope = this.authStorageService.getPermissions().grafana;
```

Comment:
Since we already have a permissions variable, this can be removed.

Reply:
ack